tests/testthat/_snaps/author.md

```diff
@@ -3,33 +3,33 @@
     Code
       challenge_legacy_author_fields()
     Message
-      x Found legacy Author and/or Maintainer field in DESCRIPTION.
-        usethis only supports modification of the Authors@R field.
+      x Found legacy 'Author' and/or 'Maintainer' field in DESCRIPTION.
+        usethis only supports modification of the 'Authors@R' field.
       i We recommend one of these paths forward:
-        * Delete these fields and rebuild with `use_author()`.
-        * Convert to Authors@R with `desc::desc_coerce_authors_at_r()`,
-          then delete the legacy fields.
+      [ ] Delete the legacy fields and rebuild with `use_author()`; or
```

Comment:
It's a little bit of a bummer that the TODO checkbox becomes `[ ]` when Unicode is not available, so it takes up more characters than the other bullets. But I really think we care more about making the output pretty in a Unicode setting.